bucket/picpick.json

```
@@ -0,0 +1,27 @@
+{
+    "version": "5.0.7",
+    "description": "A full-featured screen capture tool, Intuitive image editor, color picker, color palette, pixel-ruler, protractor, crosshair, whiteboard and more",
+    "homepage": "https://picpick.app",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://picpick.app/en/download"
+    },
+    "url": "https://www.picpick.org/releases/latest/picpick_inst.exe#/dl.7z",
+    "hash": "e790616c7069562526bcd6eda338274363a3f775fe4702f90783f48de85d0608",
+    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\uninst*\" -Force -Recurse",
+    "bin": "picpick.exe",
+    "shortcuts": [
+        [
+            "picpick.exe",
+            "PicPick"
+        ]
+    ],
+    "checkver": {
+        "url": "https://picpick.app/en/download",
+        "regex": "v([\\d.]+)<!-- -->"
+    },
+    "autoupdate": {
+        "url": "https://www.picpick.org/releases/latest/picpick_inst.exe#/dl.7z"
+    },
+    "notes": "this free version may only be used for personal use in non-commercial environments."
```

This is in license already

```suggestion
    }
```